src/main/java/com/github/hokkaydo/eplbot/module/code/CodeCommand.java

```diff
@@ -0,0 +1,208 @@
+        }
+        File submitted = createWrappedCodeAsFile(bodyStr);
+        if (submitted == null){
+            textChannel.sendMessage(Strings.getString("COMMAND_CODE_COULDNT_WRITE_THE_FILE")).queue();
+            return;
+        }
+        if (!validateFileSize(submitted)){
+            sendSubmittedAsFile(textChannel,submitted);
+            return;
+        }
+        textChannel.sendMessage(Strings.getString("COMMAND_CODE_EXCEEDED_FILE_SIZE")).queue();
+    }
+    private void performResponse(MessageChannel textChannel, String result){
+        if (!validateMessageLength(result)){
+            textChannel.sendMessage("`"+result+"`").queue();
+            return;
+        }
+        File submitted = createResponseAsFile(result);
+        if (submitted == null){
+            textChannel.sendMessage(Strings.getString("COMMAND_CODE_COULDNT_WRITE_THE_FILE")).queue();
+            return;
+        }
+        if (!validateFileSize(submitted)){
+            sendResponseAsFile(textChannel,submitted);
+            return;
+        }
+        textChannel.sendMessage(Strings.getString("COMMAND_CODE_EXCEEDED_FILE_SIZE")).queue();
+    }
+    private void sendWrappedCodeAsText(MessageChannel textChannel, String bodyStr, String codeName){
+        textChannel.sendMessage("```"+codeName.toLowerCase()+"\n"+bodyStr+"\n```").queue();
+    }
+    private boolean validateMessageLength(String content){
+        return content.length() >= 2000;
+    }
+    private File createWrappedCodeAsFile(String bodyStr){
+        try {
+            FileWriter myWriter = new FileWriter(("%s"+File.separator+"responseCode.txt").formatted(TEMP_DIR));
+            myWriter.write(bodyStr);
+            myWriter.close();
+            return new File(("%s"+File.separator+"responseCode.txt").formatted(TEMP_DIR));;
+        } catch (IOException e){
+            e.printStackTrace();
+            return null;
+        } 
+    }
+    private void sendSubmittedAsFile(MessageChannel textChannel,File serverFile){
+        FileUpload file = FileUpload.fromData(serverFile,"responseCode.txt");
+        textChannel.sendFiles(file).queue(s -> serverFile.delete());
+    }
+    private File createResponseAsFile(String result){
+        try {
+            FileWriter myWriter = new FileWriter(("%s"+File.separator+"result.txt").formatted(TEMP_DIR));
+            myWriter.write(result);
+            myWriter.close();
+            return new File(("%s"+File.separator+"result.txt").formatted(TEMP_DIR));
+        } catch (IOException e){
+            e.printStackTrace();
+            return null;
+        }
+    }
+    private void sendResponseAsFile(MessageChannel textChannel, File serverFile){
+        FileUpload file = FileUpload.fromData(serverFile,"result.txt");
+        textChannel.sendFiles(file).queue(s -> serverFile.delete());
+    }
+    private boolean validateFileSize(File file){
+        long fileSizeInBytes = file.length();
+        return fileSizeInBytes > MAX_SENT_FILE_SIZE;
```

Comment:
Condition is always inverted
=> fileSizeInBytes <= MAX_SENT_FILE_SIZE; and remove `!`s